### ISSUE TITLE
Can I automatically generate base rules?

### ISSUE THREAD

**Issue by emanrocks89** [percyliang/sempre#127]
I am new to sempre and I am trying really hard to understand its functionality and use it for my project. From what I understand, I have to write some base case rules to start, is there a way to generate those base case rules automatically? Also, is there a tutorial online that goes step by step from raw data to rules? The existing one is a little too high level and hard to follow for me.

**Comment by howcouldyouforgetthisusername** [comment]
If you mean generating a grammar for use with the (gen)overnight module, this is definitely possible, though it currently isn't implemented in SEMPRE. If you have a graph database schema with very good edge names and vertex names, you can write a little script that produces rules rules about entity types and relations. This would be a neat addition to the overnight module; plug in a database and auto-generate a lexicon and grammar, but for now you have to do it by hand.
